cylc/flow/scripts/remove.py

```diff
@@ -18,22 +18,28 @@
 
 """cylc remove [OPTIONS] ARGS
 
-Remove tasks from a running workflow and the workflow's history.
+Erase the run-history of tasks, allowing them to run again in the same flow.
```

Comment:
@oliver-sanders wasn't super-happy with this change - see this conversation: https://github.com/cylc/cylc-flow/pull/6472#discussion_r1837851623

We need a one-line description at the top. What's the best one-liner that captures both run history erasure and task pool operations? The n=0 active window is a user-facing concept, so how about this:

```suggestion
Remove tasks from the n=0 active window, and erase the run-history of past tasks.
```

Reply:
I'll write "Remove active tasks or erase the run-history of past tasks." as the tagline.

> The n=0 active window is a user-facing concept

Yes, but still somewhat advanced, so I think the tagline should avoid mentioning it.

Reply:
Unfortunately "active tasks" is ambiguous - it is usually taken to mean submitted and running. The "active window" includes various kinds of waiting tasks too.

Reply:
Ah, and super-importantly in this context - final incomplete tasks! (which may need to be removed).

Reply:
I think it's an acceptable level of ambiguity for the tagline. It's always going to be somewhat ambiguous as its a single sentence of < 80 characters. The important thing is that is clear and concise.

Reply:
I don't think "active tasks" is acceptable ambiguity - it very strongly suggests submitted and running tasks, whereas in fact we typically want to remove tasks *in the active window* that are not *active tasks* - namely waiting or final status incomplete tasks.  

IMO the "active window" concept is not advanced, or at least its not optional for users: users have to understand the active window in order to understand what they see in the UIs and what happens when they remove tasks.